test/clt-tests/core/test-freeze-unfreeze-update.rec

```diff
@@ -26,9 +26,8 @@ mysql -P9315 -h0 -e "select * from t;"
 |    1 | a    |
 +------+------+
 ––– input –––
-rm -f /tmp/update.sql; for n in `seq 1 10000`; do echo "update t set s='b' where id=1;" >> /tmp/update.sql; done; while true; do mysql -P9315 -h0 < /tmp/update.sql | break; done &
+(rm -f /tmp/update.sql; for n in `seq 1 10000`; do echo "update t set s='b' where id=1;" >> /tmp/update.sql; done; for i in `seq 1 10`; do mysql -P9315 -h0 < /tmp/update.sql; done; 2>&1 &)
 ––– output –––
-[#!/[0-9]{1}/!#] %{NUMBER}
 ––– input –––
 sleep 30; mysql -P9315 -h0 -e "select * from t;"
```

Comment:
What is this 30 sec pause for? Isn't the below supposed to be run in parallel with the updates? Can't the 30 sec pause let the above updates finish before the below starts?

Reply:
No, that's not necessary. Reduced it to 1 second